### DISCUSSION TITLE
Hystory-Log Manca nome utente su automazione

### DISCUSSION THREAD

**Discussion by teleoltre** [indomus/forum#1056] [HUB: Home Assistant]
Salve,
Se effettuo una qualsiasi cosa dall'applicazione nell'History visualizzo normalmente il nome dell'utente che ha eseguito l'azione.
Mentre se l'azione si svolge tramite una automazione, ad esempio scannerizzando una etichetta NFC, oppure premendo un pulsante su un dispositivo zigbee, viene registrato nel log come : "nome dispositivo o entità" acceso dal servizio switch.turn_on

C'è modo di sapere l'utente che ha avviato l'automazione ?
Grazie

**Comment by snakuzzo** [comment]
Le automazioni non sono avviate dall'utente (a meno che non siano eseguite a mano)

**Reply by teleoltre** [reply]
Infatti.... Se l'avvio a mano mi registra il nome.... C'è modo di ovviare al problemino ? Poichè sarebbe importante per me sapere chi ha aperto la porta scannerizzando il tag nfc. Grazie
